utilities/_utilities.viewport-heights.scss

```diff
@@ -0,0 +1,129 @@
+/* ==========================================================================
+   #VIEWPORT-HEIGHTS
+   ========================================================================== */
+
+/**
+ * inuitcss generates a series of utility classes that give a height based on
+ * viewport height unit 'vh' to whichever element they’re applied, e.g.:
+ *
+ *   <div class="u-vh--100">
```

Comment:
Hi! First of all, I really like the idea of having such utilities available by default, and I think that you came up with a quite nice solution :)

In this line you write utilities as modifiers, `u-vh--100`. However, everywhere else in the file you express these utilities as simple blocks (`u-vh-100`). I think that the former is what makes sense.

👋

Reply:
Good spot. The modifier comment is actually a mistake. As `--` distinguishes a "modification" to a block in terms of BEM but these classes are simply, well, utilities and hold single usage. 

For example, `u-vh` is not generated as a block class.